pkgs/os-specific/linux/qmk-udev-rules/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub }:
```

Comment:
Why is this not an extra output in the qmk package?

Reply:
Because it's not the way it's being done for other `-udev-rules` I could find on the packages list, and also because the build is completely different (not the same repository, not the same version, etc.)